src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -80,18 +81,18 @@
             <v-col
               cols="12"
               sm="3"
+              class="pr-4"
             >
-              <strong>Previous AGM date or a reference date</strong>
+              <strong>Most recent annual reference date or AGM date</strong>
             </v-col>
             <v-col
               cols="12"
               sm="9"
             >
               <DatePicker
-                class="pt-2 pl-4"
+                class="pt-2"
                 title="Previous AGM date or a reference date"
                 nudge-right="40"
-                :inputRules="dateRules"
```

Comment:
The input rules was causing validation on blur event for the date text field, which could occur when the user clicks on the page while the date picker is still open. I will be implementing validation using the `errorMsg` prop in the wrap-up ticket.

<img width="348" alt="Screenshot 2023-11-24 at 1 05 58 PM" src="https://github.com/bcgov/business-filings-ui/assets/122323255/9ff92583-80f6-4786-8804-433ffc2c7d4c">

Reply:
Could this component be fixed, instead, to not validate on blur?

Reply:
Yup I can fix it in the common component